crates/credentials/src/vc.rs

```diff
@@ -85,45 +86,40 @@ impl VerifiableCredential {
             vc: self.clone(),
         };
 
-        let encoded_header = header.encode()?;
-        let encoded_claims = claims.encode()?;
+        let jwt = Jwt::sign(bearer_did, key_selector, None, &claims)?;
```

Comment:
Eventually I think we will want a concept of a vcJwt as it is not exactly the same as a jwt

Reply:
for reference :

https://github.com/TBD54566975/web5-js/blob/main/packages/credentials/src/verifiable-credential.ts#L117C1-L134C4
```
  public async sign(options: VerifiableCredentialSignOptions): Promise<string> {
    const vcJwt: string = await Jwt.sign({
      signerDid : options.did,
      payload   : {
        vc  : this.vcDataModel,
        nbf : Math.floor(new Date(this.vcDataModel.issuanceDate).getTime() / 1000),
        jti : this.vcDataModel.id,
        iss : options.did.uri,
        sub : this.subject,
        iat : Math.floor(Date.now() / 1000),
        ...(this.vcDataModel.expirationDate && {
          exp: Math.floor(new Date(this.vcDataModel.expirationDate).getTime() / 1000),
        }),
      }
    });

    return vcJwt;
  }
  ```